### ISSUE TITLE
Query not existing series does not return error

### ISSUE THREAD

**Issue by xkrt** [influxdb/influxdb.com#137]
https://influxdb.com/docs/v0.9/guides/querying_data.html

>  If there was an error processing the query, the error key will be present, and will contain information explaining why the query failed. An example of this type of failure would be **attempt to query a series that does not exist**.

But InfluxDB return `200` with empty `results`.

Create db:
```
>curl -G "http://localhost:8086/query" --data-urlencode "q=CREATE DATABASE mydb"
{"results":[{}]}
```

Query not existing series:
```
>curl -G "http://localhost:8086/query" --data-urlencode "db=mydb" --data-urlencode "q=SELECT * FROM not_exist"
{"results":[{}]}
```

InfluxDB nightly (0.9.1-rc1-224-g8c715f7)

**Comment by mjdesa** [comment]
@xkrt when I run the same query

```
curl -G "http://localhost:8086/query" --data-urlencode "db=mydb" --data-urlencode "q=SELECT * FROM not_exist"
```

I get the expected result

```
{"results":[{"error":"measurement not found: \"mydb\".\"default\".not_exist"}]}
```

**Comment by beckettsean** [comment]
@xkrt @mjdesa this behavior changes with 0.9.2. There is no central authority for whether or not a measurement exists, so querying an empty or non-existent measurement will give the same results, a 204 with an empty results array. 

See https://github.com/influxdb/influxdb/issues/3341#issuecomment-123514031 for Paul's explanation.

Opened #139 to address this, waiting on a +1 and then we can merge it.

**Comment by beckettsean** [comment]
closed by #139. new docs should deploy tomorrow
